server/webapp/WEB-INF/rails.new/webpack/models/shared/plugin_infos.js

```diff
@@ -155,11 +190,13 @@ PluginInfos.PluginInfo.Authorization = function (data) {
   this.authConfigSettings = Stream(data.authConfigSettings);
   this.roleSettings       = Stream(data.roleSettings);
   this.capabilities       = Stream(data.capabilities);
-
 };
 
 PluginInfos.PluginInfo.Authorization.fromJSON = (data = {}) => new PluginInfos.PluginInfo.Authorization({
   id:                 data.id,
+  status:             PluginInfos.PluginInfo.Status.fromJSON(data.status),
+  pluginFileLocation: data.plugin_file_location,
+  bundledPlugin:      data.bundled_plugin,
```

Comment:
This is half done, and I'd left it for the next person to pick up because the plugin settings API was still in progress...